db/schema.sql

```diff
@@ -41,53 +48,65 @@ CREATE TABLE IF NOT EXISTS `ta_feedback`.`questions` (
 );
 
 CREATE TABLE IF NOT EXISTS `ta_feedback`.`dept_question_choice` (
-    `department_id` VARCHAR(50) REFERENCES `department`(`name`),
+    `department_name` VARCHAR(50),
     `term` INT NOT NULL,
-    `question_id` INT REFERENCES `questions`(`question_id`),
-    `user_id` VARCHAR(10) REFERENCES `users`(`utorid`),
+    `question_id` INT,
+    `utorid` VARCHAR(10),
     `locked` bit,
-    `position` INT NOT NULL
+    `position` INT NOT NULL,
+    FOREIGN KEY(department_name) REFERENCES department(department_name),
+    FOREIGN KEY(question_id) REFERENCES questions(question_id),
+    FOREIGN KEY(utorid) REFERENCES users(utorid)
 );
 
 CREATE TABLE IF NOT EXISTS `ta_feedback`.`course_question_choice` (
-    `survey_id` INT(0) PRIMARY KEY AUTO_INCREMENT,
```

Comment:
`survey_id` should be kept as a column, but it should be a *foreign key* to the surveys table.